dmutils/api_stubs.py

```diff
@@ -28,6 +28,42 @@ def lot(lot_id=1, slug="some-lot", name=None, allows_brief=False, one_service_li
     }
 
 
+def __as_a_service_lots():
+    return [
+        lot(lot_id=1, slug='saas', name='Software as a Service'),
+        lot(lot_id=2, slug='paas', name='Platform as a Service'),
+        lot(lot_id=3, slug='iaas', name='Infrastructure as a Service'),
+        lot(lot_id=4, slug='scs', name='Specialist Cloud Services')
+    ]
+
+
+def __cloud_lots():
+    return [
+        lot(lot_id=9, slug='cloud-hosting', name='Cloud hosting'),
+        lot(lot_id=10, slug='cloud-software', name='Cloud software'),
+        lot(lot_id=11, slug='cloud-support', name='Cloud support')
+    ]
+
+
+g_cloud_4_lots = __as_a_service_lots
```

Comment:
Are g4 - g6 likely to get used ever? I know I added something yesterday to stubs which referenced G6, but it feels like there should be a line somewhere (I don't know where it is though).

Reply:
Hopefully not, although these are here to hopefully mean there are fewer barriers to dropping these in as replacements for existing test fixtures. We can look at removing those when we're consistently using these stubs rather than text/json files.